src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -386,7 +386,7 @@ public IEnumerable<Guid> GetReferencedReleaseFileVersions(Guid releaseId, params
                 .Include(rf => rf.ReleaseFileReference)
                 .Where(rf => rf.ReleaseId == releaseId)
                 .Select(rf => rf.ReleaseFileReference)
-                .Where(rfr => types.Contains(rfr.ReleaseFileType))
+                .Where(rfr => types.Contains(rfr.ReleaseFileType) && rfr.ReplacedById == null)
```

Comment:
If ReplacedById has a value that indicates a replacement is in progress. The replacement Subject is temporary at this stage and nothing has been 'replaced' yet.

This change is excluding the original Subject, but in the content preview, until the replacement is complete, it should be the original Subject file that is shown.

This fix also needs to include making sure that it's not possible to approve a Release with a replacement in progress. By the time we get to displaying the Pre-Release, we shouldn't care about this because there should be no possibility of ReplacedById/ReplacingId having values. They get cleared when the replacement is complete.

Finally on EES-1489 there is a screenshot showing 'six' which is a subject that is deleted but still showing. This fix doesn't seem to account for that.